app/shared/text-editor/example-text-editor.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import type { ComponentStory, ComponentMeta } from "@storybook/react";
+import { ExampleTextEditor } from "./example-text-editor";
+import "./styles.css";
+import { $getRoot, $getSelection, type EditorState } from "./lexical";
+
+// When the editor changes, you can get notified via the
+// LexicalOnChangePlugin!
+const onChange = (editorState: EditorState) => {
+  editorState.read(() => {
+    // Read the contents of the EditorState here.
+    const root = $getRoot();
+    const selection = $getSelection();
+
+    console.log(root, selection);
+  });
+};
+
+export default {
+  title: "Text Editor",
+  component: ExampleTextEditor,
+} as ComponentMeta<typeof ExampleTextEditor>;
+
+const Template: ComponentStory<typeof ExampleTextEditor> = (args) => {
+  return <ExampleTextEditor {...args} />;
+};
+
+export const TextEditor = Template.bind({});
```

Comment:
```suggestion
export const TextEditor = (args) => <ExampleTextEditor {...args} />
```

Reply:
Doesn't look like anything has been updated here?

Reply:
all the CR changes are in https://github.com/webstudio-is/webstudio-designer/pull/93

Reply:
I had to merge this PR as-is and apply changes in the follow-up PR